addons/mail/static/src/models/emoji_grid_view.js

```diff
@@ -131,6 +154,8 @@ registerModel({
             },
             inverse: 'emojiGridViewAsOnScroll',
         }),
+        queuedCategoryForJump: one('EmojiPickerView.Category', {
+        }),
```

Comment:
no empty object as param, just `'.Category'),` at the end

Reply:
Would also name this field in a more declarative way, e.g. `CategorySelectedByUser`

Reply:
I just made the changes